felix/dataplane/linux/policy_mgr.go

```diff
@@ -39,21 +39,38 @@ type policyManager struct {
 	neededIPSets     map[types.PolicyID]set.Set[string]
 	ipSetsCallback   func(neededIPSets set.Set[string])
 	nftablesEnabled  bool
+	flowLogEnabled   bool
 }
 
 type policyRenderer interface {
-	PolicyToIptablesChains(policyID *types.PolicyID, policy *proto.Policy, ipVersion uint8) []*generictables.Chain
-	ProfileToIptablesChains(profileID *types.ProfileID, policy *proto.Profile, ipVersion uint8) (inbound, outbound *generictables.Chain)
+	PolicyToIptablesChains(
+		policyID *types.PolicyID,
+		policy *proto.Policy,
+		ipVersion uint8,
+		flowLogEnabled bool,
```

Comment:
It probably worth aggregating some of the arguments like `flowlogEnabled`, `staged`, `nft`, ... into a common option struct. But I'll try to do it in a separate PR.

Reply:
Since the policy rendered is an object, it would be greate to be able to create it with these things preset on the struct. But I guess we use the same renderer (which was meant to be plugable) and actually only call the funcs hangging off it :-/